static-site/src/components/Groups/Tiles/index.tsx

```diff
@@ -25,8 +26,13 @@ function createGroupTiles(groups: Group[], colors = [...theme.titleColors]): Gro
   return groups
     .sort((a, b) => a.name.localeCompare(b.name))
     .map((group) => {
-      const groupColor = colors[0]!;
-      colors.push(colors.shift()!);
+      if (colors[0] === undefined) {
+        throw new InternalError("Colors are missing.");
+      }
+      const groupColor = colors[0];
+
+      // Rotate the colors
+      colors.push(colors.shift()!);  // eslint-disable-line @typescript-eslint/no-non-null-assertion
```

Comment:
can you alleviate the need for the non-null assert by repeating the `if (colors[0] === undefined) {throw}` check prior to this?